frontends/web/src/components/language/language.test.tsx

```diff
@@ -25,8 +25,6 @@ jest.mock('react-i18next');
 const useTranslationSpy = useTranslation;
 
 describe('components/language/language', () => {
-
-
```

Comment:
I wouldn't remove the lines as there are no other changes in this file, but no strong opinon.

Reply:
Removed these spaces because other test files have no spaces between the describe and it. 🙂